### DISCUSSION TITLE
q-uploader - Use v-model for QUploader

### DISCUSSION THREAD

**Discussion by DocBouCou** [quasarframework/quasar#13741] [Ideas / Proposals]
Everything is in the title ;) .
I think that it would be much more convenient, even if for some edge or particular cases, to give access to v-model/value with q-uploader (like with q-file).

Related to: #13739 and  [#11700 ](https://github.com/quasarframework/quasar/issues/11700)

**Comment by pratik227** [comment]
@DocBouCou  But there are lots of methods available in q-uploader then why you need v-model ? If you can achieve the same using this methods.

You can use @uploading , @added etc.

**Reply by DocBouCou** [reply]
Because it is easier (more convenient) if we can simply modify the file (for instance) using v-model?

**Reply by pratik227** [reply]
I don't think it's required friendly speaking

**Reply by metalsadman** [reply]
> Because it is easier (more convenient) if we can simply modify the file (for instance) using v-model?

https://github.com/quasarframework/quasar/discussions/13722#discussioncomment-2989793
